### PR TITLE
Add basic templates for Grid

### PR DESCRIPTION
**Pull request by epetrow**

### CODE REVIEW

File: tests/_output/bootstrap/grid/grid-selection-aggregates.png
Comment (inikolova) [inline tests/_output/bootstrap/grid/grid-selection-aggregates.png:1]: In my opinion we should demonstrate row selection here as well; Also, the alt-row background color is changed

File: tests/_output/bootstrap/grid/grid-sticky-columns.png
Comment (inikolova) [inline tests/_output/bootstrap/grid/grid-sticky-columns.png:1]: Maybe we could move the selection states demonstration from the second Grid in the _grid-sticky-column-states_ test?

File: tests/_output/bootstrap/grid/grid-grouping.png
Comment (inikolova) [inline tests/_output/bootstrap/grid/grid-grouping.png:1]: Maybe we could demonstrate collapsed row as well?

**Comment by inikolova** [comment]
Defined templates look OK to me (by their purpose), however I would suggest to simplify them.

File: tests/_output/bootstrap/grid/grid-selection-aggregates.png
Comment (epetrow) [reply tests/_output/bootstrap/grid/grid-selection-aggregates.png:1]: I think visually row selection here doesn't differ from the "default" behaviour and we already have a test with row states. Also I think the current test better showcases what selection aggregates does.

File: tests/_output/bootstrap/grid/grid-grouping.png
Comment (epetrow) [reply tests/_output/bootstrap/grid/grid-grouping.png:1]: I've altered the template, so it also has a collapsed row now

Comment (Juveniel) [inline tests/_output/bootstrap/grid/grid-grouping.png:1]: grouping indicator missing from bottom two

File: tests/grid/grid-column-reordering-actions.html [hunk 1]
```diff
@@ -21,202 +21,122 @@
...
-                                                </span>
+                                                </a>
                                             </span>
-                                            <a href="#" class="k-grid-header-menu k-grid-column-menu">
-                                                <span class="k-icon k-svg-icon k-svg-i-more-vertical">
-                                                    <svg aria-hidden="true" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
-                                                        <path d="M240 128c26.4 0 48-21.6 48-48s-21.6-48-48-48-48 21.6-48 48 21.6 48 48 48m0 64c-26.4 0-48 21.6-48 48s21.6 48 48 48 48-21.6 48-48-21.6-48-48-48m0 160c-26.4 0-48 21.6-48 48s21.6 48 48 48 48-21.6 48-48-21.6-48-48-48"></path>
-                                                    </svg>
+                                        </th>
+                                        <th colspan="1" rowspan="1" class="k-filterable k-header k-sorted k-table-th">
```
Comment (Juveniel) [inline tests/grid/grid-column-reordering-actions.html:56]: check if `k-filterable` is needed here

File: tests/grid/grid-column-reordering-actions.html [hunk 2]
```diff
@@ -21,202 +21,122 @@
...
+                    <div class="k-grid-container">
+                        <div class="k-grid-content">
+                            <table class="k-grid-table k-table k-table-md">
+                                <colgroup>
+                                    <col style="width: 100px;">
+                                    <col style="width: 100px;">
+                                    <col style="width: 150px;">
+                                </colgroup>
+                                <tbody class="k-table-tbody">
+                                    <tr class="k-table-row">
```
Comment (Juveniel) [inline tests/grid/grid-column-reordering-actions.html:112]: `k-master-row` missing

File: tests/grid/grid-grouping.html [hunk 3]
```diff
@@ -339,49 +366,61 @@
...
-                                        <td class="k-table-group-td k-group-cell k-table-td"></td>
-                                        <td class="k-table-td">Chai</td>
-                                        <td class="k-table-td">30</td>
-                                        <td class="k-table-td">false</td>
-                                        <td class="k-table-td">15</td>
-                                    </tr>
-                                </tbody>
-                            </table>
-                        </div>
+                    <div class="k-grid-content">
```
Comment (Juveniel) [inline tests/grid/grid-grouping.html:369]: `k-virtual-content` class missing

File: tests/grid/grid-hierarchy.html [hunk 4]
```diff
@@ -10,114 +10,156 @@
...
+                                                <svg aria-hidden="true" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+                                                    <path d="M352 256 160 384V128z"></path>
+                                                </svg>
+                                            </span>
+                                        </td>
+                                        <td class="k-table-td">1</td>
+                                        <td class="k-table-td">Beverages</td>
+                                        <td class="k-table-td">Soft drinks, coffees, teas, beers and ales</td>
+                                    </tr>
+                                    <tr class="k-master-row k-alt k-table-row k-table-alt-row">
```
Comment (Juveniel) [inline tests/grid/grid-hierarchy.html:75]: `k-expanded` missing

File: tests/grid/grid-sticky-columns.html [hunk 5]
```diff
@@ -10,330 +10,334 @@
...
+                            <colgroup>
+                                <col style="width: 250px;">
+                                <col style="width: 150px;">
+                                <col style="width: 200px;">
+                                <col style="width: 250px;">
+                                <col style="width: 250px;">
+                            </colgroup>
+                            <thead class="k-table-thead">
+                                <tr class="k-table-row">
+                                    <th colspan="1" rowspan="1" class="k-grid-content-sticky k-header k-table-th" style="left: 0px; right: 250px; border-right-width: 1px;">
```
Comment (Juveniel) [inline tests/grid/grid-sticky-columns.html:26]: `k-grid-header-sticky` instead of `k-grid-content-sticky`

**Review by Juveniel — APPROVED** [review]

File: tests/_output/bootstrap/grid/grid-column-resizing-actions.png
Comment (inikolova) [inline tests/_output/bootstrap/grid/grid-column-resizing-actions.png:1]: Grid top border is missing -> could you please check?

**Review by inikolova — APPROVED** [review]

File: tests/_output/bootstrap/grid/grid-column-resizing-actions.png
Comment (epetrow) [reply tests/_output/bootstrap/grid/grid-column-resizing-actions.png:1]: I checked and it seems like it's a glitch in the visual tests, when you open the tests locally the border is there in both Chrome and Firefox

**Pull request merged**